Formula/vowpal-wabbit.rb

```diff
@@ -3,10 +3,9 @@ class VowpalWabbit < Formula
   homepage "https://github.com/VowpalWabbit/vowpal_wabbit"
   # pull from git tag to get submodules
```

Comment:
By the way, since it looks like your submodules are for `fmt`, `spdlog`, and `rapidjson`, and we pass `*_SYS_DEP` to `cmake`: can we avoid a git clone here? Tarballs are faster and much better for caching.